setup.py

```diff
@@ -35,7 +35,7 @@ def get_version(*file_paths):
     keywords='edx codejail',
     author='edX',
```

Comment:
@nedbat should this "author" change at any point? I have no idea and no formed opinion.